pkg/strutil/convert.go

```diff
@@ -24,3 +25,23 @@ func NoCopyBytesToString(b []byte) string {
 func NoCopyStringToBytes(s string) []byte {
 	return *(*[]byte)(unsafe.Pointer(&s))
 }
+
+func ToStrSlice[T uint64 | uint | int64 | int](input []T, withQuote ...bool) []string {
+	result := make([]string, len(input))
+	if len(withQuote) == 0 {
+		withQuote[0] = false
```

Comment:
panic here if nothing passed in.

Reply:
<img width="682" alt="image" src="https://github.com/erda-project/erda/assets/13919034/cf900ce1-9b78-49cc-a5fa-0e1bd729d7a7">